packages/client/tests/functional/_utils/relationMode/computeMatrix.ts

```diff
@@ -0,0 +1,69 @@
+import { Providers } from '../providers'
+
+type ComputeMatrix = {
+  relationMode: string
+  providersDenyList?: Providers[]
+}
+
+export function computeMatrix({ relationMode, providersDenyList }: ComputeMatrix) {
+  const providersBase = [
+    Providers.POSTGRESQL,
+    Providers.COCKROACHDB,
+    Providers.SQLSERVER,
+    Providers.MYSQL,
+    Providers.SQLITE,
+  ] as const
+  //  Note SetDefault is not implemented in the emulation
```

Comment:
Still true?

Reply:
Yes, it was never implemented and same today
https://github.com/prisma/prisma-engines/blob/810aaacead8837bb6f5b6a051d719353d2c360fb/psl/psl-core/src/datamodel_connector/relation_mode.rs#L18-L23